### ISSUE TITLE
Learning rate and optimizer

### ISSUE THREAD

**Issue by yunfei-teng** [akamaster/pytorch_resnet_cifar10#8]
Could you provide the optimizer you were using for the training including learning rates and momentum, please? My test error only reaches 20% on resnet20.

**Comment by akamaster** [comment]
the repo is self contained, take a look at https://github.com/akamaster/pytorch_resnet_cifar10/blob/master/run.sh and https://github.com/akamaster/pytorch_resnet_cifar10/blob/master/trainer.py

**Comment by p3i0t** [comment]
Simply using optimizer Adam with learning rate _lr=0.001_(other hyperparameters unchanged), I get <5% error rate on CIFAR10 test set on resnet20 after training ~60 epochs.
